include/seqan3/alignment/pairwise/execution/execution_handler_sequential.hpp

```diff
@@ -52,27 +52,18 @@ namespace seqan3::detail
 class execution_handler_sequential
 {
 public:
-    /*!\name Constructors, destructor and assignment
-     * \{
-     */
-    execution_handler_sequential()                                                 = default;
-    execution_handler_sequential(execution_handler_sequential const &)             = default;
-    execution_handler_sequential(execution_handler_sequential &&)                  = default;
-    execution_handler_sequential & operator=(execution_handler_sequential const &) = default;
-    execution_handler_sequential & operator=(execution_handler_sequential &&)      = default;
-    ~execution_handler_sequential()                                                = default;
-    //!}
```

Comment:
why are these removed? If this class is supposed to be an aggregate, please make it a struct